staff/gerard-celma/mailchimp-template/style.sass

```diff
@@ -0,0 +1,205 @@
+@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro|Raleway')
+
+// Color Palette
+$blue: #46BDD5
+$light-grey: #F0F0F0
+$mid-grey: #CCD0CF
+$mid-dark-grey: #3F3F37
+$dark-grey: #181a1c
+$orange: #FF664D
+$light-orange: #ff8668
+
+// Mixins
```

Comment:
Mixins are used to reuse code if value changes. If it does not change, extend/inheritance should be used.

Reply:
Very good one, didn’t knew this, i’ll make the change, thank you Alexander Beard 👏🏼👏🏼👏🏼